participations/info.yml

```diff
@@ -0,0 +1,86 @@
+---
+query:
+  model: true
+  sql: |
+    -- /!\ Asked by city : not taking into account comments, endorsments and proposal votes in contributions !
```

Comment:
remove comment

Reply:
Fixed in [db95ca7](https://github.com/OpenSourcePolitics/decidim_cards/pull/42/commits/db95ca79aa1e2275149b39294497cb23cebe7dbf)